app/assets/stylesheets/alchemy/_mixins.scss

```diff
@@ -110,3 +110,19 @@
     line-height: 8px;
   }
 }
+
+@mixin linked-button($border-radius: $default-border-radius) {
+  @include sassy-button(
+    'matte',
+    $border-radius,
+    $sb-font-size,
+    #ffd77a,
```

Comment:
Color literals like `#ffd77a` should only be used in variable declarations; they should be referred to via variable everywhere else.